src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -68,28 +68,34 @@ const DataViewTable: React.FC = () => {
     return <Message error header={strings.ERROR_GENERIC} content={error.message} />
   }
 
+  const searchComponent =
+    dataViewTable && dataViewTable?.searchFields?.length > 0 ? (
+      <Search
+        placeholder={strings.DATA_VIEW_SEARCH_PLACEHOLDER}
+        onSearchChange={(e: any) => {
+          setSearchText(e.target.value)
+          setDebounceInput(e.target.value)
+        }}
+        input={{ icon: 'search', iconPosition: 'left' }}
+        open={false}
+        value={searchText}
+      />
+    ) : null
```

Comment:
The placement of the "Search" box relative to the Title and Filter menus needed to change depending on whether it's a lookup table or database view. (Lookup table doesn't show the Title, since the page already has its own title). So moved the "Search" element to its own component for easier re-use.